README.md

```diff
@@ -301,6 +301,16 @@ Browsershot::url('https://example.com')
    ...
 ```
 
+#### Ignore HTTPS errors
+
+You can ignore HTTPS errors, if necessary.
+
+```php
+Browsershot::url('https://example.com')
+   ->ignoreHttpsErrors()
+   ...
+```
+
```

Comment:
Small addition to the README. I'm not sure how to best explain what causes the issue, so this may need some tweaking.
Also, I use `trim_trailing_white_space_on_save` which, I believe, explains why some of the other lines were modified.